test/conservation_checker_tests.jl

```diff
@@ -82,17 +84,29 @@ for FT in (Float32, Float64)
         )
 
         # set non-zero radiation and precipitation
-        F_r = cf.radiative_energy_flux_toa
+        F_r = 200 .* CC.Fields.ones(space)
         P = cf.P_liq
         Δt = float(cs.Δt_cpl)
 
+        volume = integral(ones(space))
+        # 0.5 is the area fraction
+        water_from_precipitation = integral(P) .* FT(Δt) .* FT(0.5)
+        energy_from_radiation = integral(F_r) .* FT(Δt)
+        energy_per_unit_cell = CC.Fields.ones(space) .* energy_from_radiation ./ volume
+        water_per_unit_cell = CC.Fields.ones(space) .* water_from_precipitation ./ volume
+
         # analytical solution
-        tot_energy_an = sum(FT.(F_r .* 3Δt .+ 1e6 .* 1.25))
-        tot_water_an = sum(FT.(.-P .* 3Δt .* 0.5 .+ CC.Fields.ones(space)))
+        # 1.25 comes from the area fraction of the ocean
```

Comment:
why do we include the ocean area fraction but not ice or land?

Reply:
Because there's water only on the ocean as the `get_field` are defined here (and there's no ice in this test).